app/Actions/Albums/Extensions/PublicIds.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace App\Actions\Albums\Extensions;
+
+use AccessControl;
+use App\Models\Album;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Collection as BaseCollection;
+use Illuminate\Support\Facades\DB;
+
+class PublicIds
+{
+	/** @var BaseCollection */
+	private $white_list = null;
+	/** @var BaseCollection */
+	private $black_list = null;
```

Comment:
These terms are not considered politically-correct in the US anymore :smiley:.

Reply:
Hahahaha I knew this would raise an issue. :laughing:

Reply:
Fixed.